<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModbusModule</class>
 <widget class="QWidget" name="ModbusModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QSpinBox
{
	max-height:20px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="7" column="0" colspan="2">
    <widget class="QCheckBox" name="cbShowCommands">
     <property name="text">
      <string>显示通信指令</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbAddr">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>设备地址（1-255）：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lbBuad">
     <property name="text">
      <string>波特率：</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QPushButton" name="pbSave">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>保存并打开端口</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cbBuad">
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lbData">
     <property name="text">
      <string>数据位：</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cbData">
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lbStop">
     <property name="text">
      <string>停止位：</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="cbStop">
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="cbParity">
     <item>
      <property name="text">
       <string>NONE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ODD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EVEN</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lbParity">
     <property name="text">
      <string>校验位：</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="lbInfo_3">
       <property name="text">
        <string>发送：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="lbInfo">
       <property name="text">
        <string>未收到数据！</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lbSend">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lbInfo_2">
       <property name="text">
        <string>接收：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lbRecv">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="sbAddr">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>外部通信端口设置：</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
